Sources/UI/Extensions/Image+PovioKit.swift

```diff
@@ -8,13 +8,11 @@
 
 import SwiftUI
 
-@available(iOS 14, *)
```

Comment:
PovioKit currently supports iOS 13+. We'll probably bump soon to 14. Can we leave this intact?

Reply:
@borut-t What do you mean by initials? Can you please give an example

Reply:
We can either set the image or initials inside the profile image view. Usually, we do this as a fallback when the user didn't provide picture.

Reply:
We never had the implementation to set the initial properties. Was inspired by SwiftUI's `Image` as it has a simple init and then modifiers to add more specific requirements.
I can add nullable parameters to the init if we want to provide them from the get-go.
Personally wouldn't prefer it just because of style but I don't have strong feelings about this. Let me know what you think.

Reply:
I don't like that either. Maybe we've miss-understood. I just want to provide second option to image.